tracer/test/test-applications/integrations/Samples.GraphQL4/Samples.GraphQL4.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">4.6.1</ApiVersion>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">5.1.1</ApiVersion>
```

Comment:
nit-we could rename the sample to `Samples.GraphqlGT4` to make it obvious it handles not only v4 but greater version?

Reply:
Yes I thought the exact same thing when writing it. Note the use of both "nit" and the conditional :p